bundles/org.openhab.ui/web/src/assets/definitions/widgets/actions.js

```diff
@@ -122,6 +122,10 @@ export const actionParams = (groupName, paramPrefix) => {
         return ['variable'].indexOf(configuration[paramPrefix + 'action']) >= 0
       }),
     pt(paramPrefix + 'actionVariableValue', 'Variable Value', 'The value to set the variable to').a()
+      .v((value, configuration, configDescription, parameters) => {
+        return ['variable'].indexOf(configuration[paramPrefix + 'action']) >= 0
+      }),
+    pt(paramPrefix + 'actionVariableKey', 'Variable Key', 'The value to set the key of avariable').a()
```

Comment:
Still not satisfied with the description here, it needs to be more detailed:

```suggestion
    pt(paramPrefix + 'actionVariableKey', 'Variable Key', 'Consider the variable value is an object and set the correponding deep property within that object using a key syntax. Examples: <code>user.name</code>, <code>user[0].address[1].street</code>, <code>[0]</code>, <code>[0].label</code>. The inner property and its parent hierarchy will be created if missing.').a()
```

Also remember to regenerate the components' documentation.

Reply:
done, also updated the docs